packages/magento-product/components/AddProductsToCart/AddProductsToCartSnackbar.tsx

```diff
@@ -35,8 +36,20 @@ export function AddProductsToCartSnackbar(props: AddProductsToCartSnackbarProps)
 
   const items = filterNonNullableKeys(data?.addProductsToCart?.cart.items)
 
+  const productSkus: (string | undefined | null)[] = []
+
+  submittedVariables?.cartItems.forEach((item) => {
+    productSkus.push(item.sku)
+  })
+
+  const productsAdded = items
+    .filter((item) => productSkus.includes(item.product.sku))
+    .map((product) => product.product.name)
```

Comment:
I've got a feeling we can make this easier by not creating a productSkus directly but by using 'find' on the submittedVariables?.cartItems?